### PR TITLE
Known issue update [SKIP CI]

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
- Adding an entry to known issue as discussed at https://github.com/vmware/docker-volume-vsphere/issues/2006#issuecomment-348403054

- Adding an entry for Windows Plugin performance improvement issue.

### CODE REVIEW

File: docs/external/known-issues.md [hunk 1]
```diff
@@ -5,11 +5,22 @@ title: Known Issues
 This section lists the major known issues with vSphere Docker Volume Service. For a complete list of issues, please check our Github issues(https://github.com/vmware/docker-volume-vsphere/issues) page. If you notice an issue not listed in Github issues page, please do file a bug on the [Github repo](https://github.com/vmware/docker-volume-vsphere/issues)
 
 -  Volume metadata file got deleted while removing volume from VM(placed on Esx2) which is in use by another VM(placed on Esx1) [#1191](https://github.com/vmware/docker-volume-vsphere/issues/1191). It's an ESX issue and will be available in the next vSphere release.
--  Full volume name with format like "volume@datastore" cannot be specified in the compose file for stack deployment. [#1315](https://github.com/vmware/docker-volume-vsphere/issues/1315). It is a docker compose issue and a workaround has been provided in the issue.
 -  Volume creation using VFAT filesystem is not working currently. [#1327](https://github.com/vmware/docker-volume-vsphere/issues/1327)
```
Comment (tusharnt) [inline docs/external/known-issues.md]: Not sure whether any customers care about VFAT. Can we please remove this entry if everyone agrees it is not frequently asked feature?

File: docs/external/known-issues.md [hunk 2]
```diff
@@ -5,11 +5,22 @@ title: Known Issues
...
 -  Plugin fails to create volumes after installation on VM running boot2docker Linux. This is because open-vm-tools available for boot2docker doesn't install the vSockets driver and hence the plugin is unable to contact the ESX service. [#1744](https://github.com/vmware/docker-volume-vsphere/issues/1744)
```
Comment (tusharnt) [inline docs/external/known-issues.md]: #1744 is closed saying that documentation is updated. Why is this issue still in the list then?

File: docs/external/known-issues.md [hunk 3]
```diff
@@ -5,11 +5,22 @@ title: Known Issues
...
 -  Currently "vmdk-opsd stop" just stops (exits) the service forcefully. If there are operations in flight it could kill them in the middle of execution. This can potentially create inconsistencies in VM attachement, KV files or auth-db. [#1073](https://github.com/vmware/docker-volume-vsphere/issues/1073)
 -  vDVS volume that has already been created cannot be resized or it's size cannot be updated.
+- If there are volumes with the same name on different DS then the long name has to be given (e.g. `myVolume@vsanDatastore`). Let's take an example as shown below, full volume name should be passed while mounting the volume.
```
Comment (tusharnt) [inline docs/external/known-issues.md]: DS -> datastores
the long name has to be given -> use the long name 

Why do we need to explain this in detail here? It belongs in the issue which is already linked here.

File: docs/external/known-issues.md [hunk 4]
```diff
@@ -5,11 +5,22 @@ title: Known Issues
...
+```
+root@sc-rdops-vm02-dhcp-52-237:~# docker volume ls
+DRIVER              VOLUME NAME
+vsphere:latest      myVolume@sharedVmfs-0
+vsphere:latest      myVolume@vsanDatastore
+```
+
+## Windows Plugin Known issues
+
+- Slowness is detected while creating vsphere volume from Windows VM. [#2010](https://github.com/vmware/docker-volume-vsphere/issues/2010)
```
Comment (tusharnt) [inline docs/external/known-issues.md]: Slowness is detected while creating vsphere volume from Windows VM -> Volume creation initiated by Windows Container Hosts is slow.

File: docs/external/known-issues.md (hunk 1 above)
Comment (shuklanirdesh82) [reply docs/external/known-issues.md]: yeah we can remove this.

File: docs/external/known-issues.md (hunk 3 above)
Comment (shuklanirdesh82) [reply docs/external/known-issues.md]: Let me add this to FAQ instead of the known issue page.

**Pull request merged**